OneCommunityGlobal/HGNRest
# Ivy-send-management-email-deactivedUser

# Description
<img width="757" alt="Screenshot 2024-07-18 at 5 10 21 PM" src="https://github.com/user-attachments/assets/c2ae04e6-9fc6-42b4-82fe-e55cfe192ef0">
<img width="778" alt="Screenshot 2024-07-22 at 4 16 26 PM" src="https://github.com/user-attachments/assets/9fac6996-eeec-43ff-82fa-4b7541804e7f">
Or Implements # (WBS) 
## Main changes explained:
- Update src/controllers/userProfileController.js for synchronized deactivated email sender
- Update src/helpers/userHelper.js to introduce a new function and correct typo

…

## How to test:
1.check into current branch
2. Gmail API
Before testing, you need to set up your own test email id to send the mail notification.
This is only for test purposes. Once tested you can revert these changes..
To send an email using the Gmail API with OAuth2 and nodemailer, you can just follow the video below and set it up.
https://www.youtube.com/watch?v=-rcRf7yswfM&t=10s&ab_channel=MafiaCodes

Fill in the these variable values from the above setup in the .env file of the backend.
REACT_APP_EMAIL =
REACT_APP_EMAIL_CLIENT_ID =
REACT_APP_EMAIL_CLIENT_SECRET =
REACT_APP_EMAIL_REFRESH_TOKEN =
REACT_APP_EMAIL_CLIENT_REDIRECT_URI = https://developers.google.com/oauthplayground
sendEmail = true
3. do `npm install` and `...` to run this PR locally
4. Clear site data/cache
5. log as admin/owner/manager user
6. verify function:

https://github.com/user-attachments/assets/3373ca70-5ad7-4c88-be95-cddd75330cb6

https://github.com/user-attachments/assets/7fb0041e-2c81-435c-82fb-7a8d359c3d37




## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
